src/sql/plot.py

```diff
@@ -7,18 +7,20 @@
 
 try:
     import matplotlib.pyplot as plt
+    from matplotlib.colors import Normalize
```

Comment:
since matplotlib is optional, let's add the requires decorator to the `ggplot` object so it raises an error asking to install matplotlib: https://ploomber-core.readthedocs.io/en/latest/dependencies.html